src/WheelsAndGoods.Application/Services/OrdersService.cs

```diff
@@ -46,5 +46,27 @@ public async Task<IReadOnlyCollection<OrderResponse>> GetOrders()
             var orders = await _unitOfWork.Orders.GetOrders();
             return _applicationMapper.MapCollectionOrEmpty(orders, _applicationMapper.ToOrderResponse);
         }
+
+        public async Task<OrderResponse> UpdateOrder(UpdateOrderRequest updateOrderRequest, Guid orderId)
+        {
+            var order = await _unitOfWork.Orders.GetById(orderId, false);
```

Comment:
With approach from previous comment we can just enable tracking here and then instead of update call in transaction just map and save:
```
            await _unitOfWork.CommitTransactionAsync(() =>
            {
                     _applicationMapper.MapToOrder(updateOrderRequest, order);
            });
```